pysat/tests/test_meta_header.py

```diff
@@ -142,3 +162,29 @@ def test_init_metaheader_in_meta(self, header_data):
         # Ensure both initialization methods work the same
         assert meta.header == self.meta_header
         return
+
+    def test_get_metaheader_in_meta(self):
+        """Test MetaHeader attribute retrieval from Meta."""
+        # Initalize the header data through the meta object
+        test_attr = "PI"
+        test_val = "pysat development team"
+        meta = pysat.Meta(header_data={test_attr: test_val})
+
+        # Test value retrieval from Meta and MetaHeader
+        assert getattr(meta.header, test_attr) == test_val
```

Comment:
Does `meta.header.PI` also work?

Reply:
Yes, that's the formulation that `getattr` tests.